Na minha tarefa ao executar o comando de 'testa.sh comando_ver_sessoes', todas as funções
do arquivo já estavam sendo chamadas corretamente e apenas encontrei 2 erros com assert 
na função que simulava a criação de sessões de usuários.

Na primeira delas, o usuário deveria ser um administrador do sistema, mas o ID usado 
era o de um usuário comum (S-00000004), como foi verificado na lista de usuários.

No segundo erro era o caso oposto, o teste deveria simular um usuário comum, mas o ID
utilizado era do administrador (S-00000002).

A única alteração que foi necessária executar no arquivo 'comando_ver_sessoes_TST.py'
foi a troca desses 2 IDs, passando o ID de administrador no primeiro caso, 
e o ID de usuário comum para o segundo teste e, assim, os testes passaram 
sem mais nenhum erro.